code/modules/atmospherics/machinery/pipes/layermanifold.dm

```diff
@@ -27,16 +27,12 @@
 	return ..()
 
 /obj/machinery/atmospherics/pipe/layer_manifold/proc/nullifyAllNodes()
-	var/list/obj/machinery/atmospherics/needs_nullifying = get_all_connected_nodes()
+	for(var/obj/machinery/atmospherics/A in nodes)
```

Comment:
Are these changes somehow related to the debug verb ?

Reply:
yes. manifolds didnt populate its nodes list, which resulted in false one-way connection detections